Resources/public/Admin.js

```diff
@@ -103,6 +103,31 @@ var Admin = {
                 checkboxClass: 'icheckbox_square-blue',
                 radioClass: 'iradio_square-blue'
             });
+
+            var previousIndex;
+
+            jQuery('tbody .iCheck-helper', subject).click(function (evt) {
```

Comment:
Not necessarily. `function setup_checkbox_selection_range(subject) {}` could work too, right? If yes, not sure what that would do to the global scope. If not or if it pollutes the global scope, do what you proposed.